### DISCUSSION TITLE
Release Date [Jan 27]

### DISCUSSION THREAD

**Discussion by dstala** [nocodb/nocodb#5022] [Discord Threads]
> 🙋‍♂️ **davidpfahler**

**Comment by o1lab** [comment]
> 🙋‍♂️ **wkw**  
#### we have a daily build on the current develop branch. You can test it on that image.   
####    
#### ```   
#### nocodb/nocodb-daily:latest   
#### ```  
#  
> 🙋‍♂️ **davidpfahler**  
#### Thanks!  
#  
> 🙋‍♂️ **davidpfahler**  
#### No joy, unfortunately  
#  
> 🙋‍♂️ **wkw**  
#### which issue are u facing?  
#  
> 🙋‍♂️ **davidpfahler**  
#### https://discord.com/channels/661905455894888490/661905455894888493/1053593150263934987  
#  
> 🙋‍♂️ **davidpfahler**  
#### It's very hard to debug / show, b/c I cannot disclose production data and the issue seems tightly connected to some data, but I cannot figure out what the problem is  
#  
> 🙋‍♂️ **wkw**  
#### any useful logs from backend side?  
#  
> 🙋‍♂️ **wkw**  
#### does the problematic table contain formula field?  
#  
> 🙋‍♂️ **davidpfahler**  
#### The logs dont show any errors. The API response is 200, but returns an empty list.  
#  
> 🙋‍♂️ **davidpfahler**  
#### No formula fields in any table  
#  
> 🙋‍♂️ **wkw**  
#### any LTAR columns? or can u share wt kinds of columns that u hv?  
#  
> 🙋‍♂️ **davidpfahler**  
#### Are there some kind of metadata tables I could share that would be useful?  
#  
> 🙋‍♂️ **davidpfahler**  
#### The structure is as follows: Invoices table has an LTAR column that links an invoice row to an offer row. The Offers table has a lookup on the Invoice's status column and an LTAR column linking to a deal. The Deals table has also a Lookup on the invoice's status and two LTAR columns linking to an Account and a Contact. Also, rows in the Tasks table have an LTAR column linking a Task to a Deal.  
#  
> 🙋‍♂️ **davidpfahler**  
#### <@495466997304262658> I found the source of the problem. There is a file column in the invoices table. This column is apparently supposed to hold JSON arrays, but some fields have a weird, JavaScript object like structure that cannot be parsed. As this database was never touched by anything else, nocodb must have at some point introduced these and (now?) cannot handle them (anymore?).  
#  
> 🙋‍♂️ **wkw**  
#### can u create an issue on github  
#  
> 🙋‍♂️ **davidpfahler**  
#### https://github.com/nocodb/nocodb/issues/4709  
#  

 [Autogenerated from discord](https://discord.com/channels/661905455894888490/1055048039162708028)  
- - - -  
#### Join [NocoDB's](http://www.nocodb.com/)  community  
[Discord](https://discord.gg/5RgZmkW) • [Twitter](https://twitter.com/nocodb) • [Reddit](https://www.reddit.com/r/NocoDB/) • [Docs](https://docs.nocodb.com/)
